cli/server.go

```diff
@@ -111,6 +111,8 @@ func Server(newAPI func(*coderd.Options) *coderd.API) *cobra.Command {
 		autoImportTemplates              []string
 		spooky                           bool
 		verbose                          bool
+		// maxWorkspacesPerUser is a uint to ensure a number > 0.
+		maxWorkspacesPerUser uint
```

Comment:
Do we want to support a flag like this for a long time? It might be best to mark experimental, because it seems unlikely we'd keep this mechanism long-term.

Reply:
We will re-add this control in the quota system, and I think we only want one way to configure max workspaces to avoid confusion. So, I think we're better off hard-coding a max for DoS protection and avoiding configuration for now.